drivers/mysql/mysql.go

```diff
@@ -148,7 +147,7 @@ func (driver *mysql) Lock() error {
 	// This will wait until the lock can be acquired or until the statement timeout has reached.
 	query := fmt.Sprintf("SELECT GET_LOCK(?, %d)", driver.config.StatementTimeoutInSecs)
```

Comment:
Unrelated: we aren't actually using this method right? Maybe remove it?

Reply:
Actually we are still using it. Intentionally left it if they don't use the `morph.WithLock`.